pom.xml

```diff
@@ -127,6 +127,17 @@
       <artifactId>guava</artifactId>
       <version>21.0</version>
     </dependency>
+    <dependency>
+      <groupId>com.google.auto.value</groupId>
+      <artifactId>auto-value-annotations</artifactId>
+      <version>1.6.6</version>
+    </dependency>
+    <dependency>
+      <groupId>com.google.auto.value</groupId>
```

Comment:
adding value (objects), much appreciated!

Reply:
it's a great library, thanks for introducing me to it!